include/seqan3/search/dream_index/interleaved_bloom_filter.hpp

```diff
@@ -402,46 +353,21 @@ class interleaved_bloom_filter
     /*!\name Lookup
      * \{
      */
-    /*!\brief Determines set membership of a given value.
-     * \param[in] value The raw value to process.
-     *
-     * \attention The result of this function must always be bound via reference, e.g. `auto &` to prevent copying.
+    /*!\brief Returns seqan3::interleaved_bloom_filter::bulk_agent to be used for lookup.
+     * \attention Calling seqan3::interleaved_bloom_filter::increase_bin_number_to invalidates all
+     * seqan3::interleaved_bloom_filter::bulk_agent constructed for this Interleaved Bloom Filter.
      *
      * \details
      *
      * ### Example
      *
-     * \include test/snippet/search/dream_index/interleaved_bloom_filter_bulk_contains.cpp
+     * \include test/snippet/search/dream_index/bulk_agent_constructor.cpp
```

Comment:
The name of the snippet is a bit misleading, but it is okay for me :)

Maybe `bulk_agent_construction.cpp`